src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/ccd/datamigration/controller/CcdDataMigrationController.java

```diff
@@ -40,16 +40,23 @@ public CallbackResponse migrate(
         boolean shouldMigrateCase = shouldMigrateCase(caseData);
         log.info("shouldMigrateCase >>> {}", shouldMigrateCase);
         if (shouldMigrateCase) {
-            caseData.remove(STATE);
+            String value = Objects.toString(caseData.get(JUDGE_ALLOCATED));
+            caseData.put(JUDGE_ALLOCATED, new String[] { value } );
+
             return AboutToStartOrSubmitCallbackResponse.builder().data(caseData).build();
         } else {
             return AboutToStartOrSubmitCallbackResponse.builder().build();
         }
     }
 
     private boolean shouldMigrateCase(Map<String, Object> caseData) {
-        String state = Objects.toString(caseData.get(STATE), "");
-        log.info("state >>> {}", state);
-        return !isEmpty(state);
+        Object judgeAllocated = caseData.get(JUDGE_ALLOCATED);
+        if (nonNull(judgeAllocated) && !ObjectUtils.isEmpty(judgeAllocated)) {
+            if (judgeAllocated instanceof String) {
```

Comment:
just a thought - this could be written as `return judgeAllocated instaceof String`